spec.html

```diff
@@ -469,8 +469,8 @@ <h1>%Iterator.prototype%.filter ( _filterer_ )</h1>
         <h1>%Iterator.prototype%.take ( _limit_ )</h1>
         <emu-alg>
           1. Let _iterated_ be ? GetIteratorDirect(*this* value).
-          1. Let _integerLimit_ be ? ToInteger(_limit_).
-          1. If _integerLimit_ &lt; 0, throw a *RangeError* exception.
+          1. Let _integerLimit_ be ? ToIntegerOrInfinity(_limit_).
+          1. If _integerLimit_ &lt; 0 or is equal to +∞, throw a *RangeError* exception.
```

Comment:
It seems useful for a variable to be passed into “take”, and for Infinity to mean “take everything” - same on drop.